index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Apple</title>
+  <link href="styles/style.css" rel="stylesheet" type="text/css">
+  <link rel="icon" href="img/apple-icon.ico" />
+  <meta charset="utf-8"/>
+</head>
+
+<body>
+  <div class="top-content">
+    <div class="flex-spacer-1"></div>
+    <nav class="flex-spacer-1">
+      <ul>
+        <li>
+          <div><button class="round-first-button"><img class="button-apple-logo" src="img/globalnav_text.svg" alt="apple-logo"></button>
+          </div>
+        </li>
+        <li>
+          <div><button><img class="button-store" src="img/globalnav_text.svg" alt="store"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-mac" src="img/globalnav_text.svg" alt="mac"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-ipod" src="img/globalnav_text.svg" alt="ipod"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-iphone" src="img/globalnav_text.svg" alt="iphone"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-ipad" src="img/globalnav_text.svg" alt="ipad"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-itunes" src="img/globalnav_text.svg" alt="itunes"></button></div>
+        </li>
+        <li>
+          <div><button><img class="button-support" src="img/globalnav_text.svg" alt="support"></button></div>
+        </li>
+        <li class="last-list-element"><input></li>
+      </ul>
+    </nav>
+    <div class="flex-spacer-3">
+      <img class="flex-spacer-1" src="img/your_verse_title_large.png" alt="your verse">
+      <img class="flex-spacer-1" src="img/your_verse_subhead_large.png" alt="your verse subhead">
+      <div class="flex-spacer-1 third-title-hider"><img src="img/cta_your_verse_46pt.png" alt="your verse video"></div>
+    </div>
+    <div class="flex-spacer-1"></div>
+  </div>
+  <div class="bottom-content bottom-column-1"></div>
+  <div class="bottom-content bottom-column-2"></div>
+  <div class="bottom-content bottom-column-3"></div>
+  <div class="bottom-content bottom-column-4"></div>
```

Comment:
- [ ] These images should be wrapped in an `a` tag because they are links.[(link to page)](https://web.archive.org/web/20140301004610/http://www.apple.com/)

Reply:
Completed the change as requested.